api/capi/include/nnstreamer-single.h

```diff
@@ -25,10 +25,8 @@
  * @detail This is targetting Tizen 5.5 M2.
  */
 
-#ifndef __NNSTREAMER_CAPI_SINGLE_H__
-#define __NNSTREAMER_CAPI_SINGLE_H__
-
-#include "nnstreamer.h"
+#ifndef __TIZEN_MACHINELEARNING_NNSTREAMER_SINGLE_H__
+#define __TIZEN_MACHINELEARNING_NNSTREAMER_SINGLE_H__
 
```

Comment:
Is it ok to remove ```#include "nnstreamer.h"```?

E.g., ```ml_tensors_info_s``` is not defined in nnstreamer-single.h

Reply:
If `#include "nnstreamer.h"` is used, below error occurs.

```bash
include/nnstreamer-single.h:31: ERROR:
  Local includes ("name.h") must not be used in public headers.
```

One of the possible solutions is use `#include <nnstreamer.h>`